moleculekit/tools/crystalpacking.py

```diff
@@ -7,26 +7,31 @@
 # -> Notes: line 69 explanation missing
 
 from moleculekit.molecule import Molecule
+from moleculekit.readers import _getPDB
 import numpy as np
 import copy
-import os
+from tqdm import tqdm
 import logging
+import unittest
 logger = logging.getLogger(__name__)
 
 
 def _get_cellloc(center, angles, box_size):
-    '''Computes the Unit Cell's location of the molecule object.
+    """
+    Computes the Unit Cell's location of the molecule object.
     Takes as argument the Euler angles of the Unit Cell's sides and performs
     a small correction to account for oblique sides.
-    Allows to correct x position dependent on y and z.'''
+    Allows to correct x position dependent on y and z.
+    """
     change = np.zeros(3)
     change[0] = center[1] / np.tan(angles[2]) + center[2] / np.tan(
         angles[1])  # get min_value x unit cell at given y,z
     return np.floor((center - change) / box_size)  # correct x axis boundaries according to angles
 
 
 def _is_outside(center, cellloc):
-    '''Checks if a molecule object is inside the defined unit cell.
+    """
```

Comment:
In this case they are private docs so whatever. But keep in mind that comments on the first line of """ are added as short descriptions in sphinx and the text after the first line is added to the full docs. So please don't go around changing those

Reply:
OK, I won't change for the private docs back, but thanks for the explanation. I will do it on the non-private one.